session/session.go

```diff
@@ -321,28 +321,29 @@ func (s *Session) handleTick(ctx context.Context) {
 	s.wm.WantBlocks(ctx, live, nil, s.id)
 
 	// do no find providers on consecutive ticks
-	// -- just rely on periodic rebroadcast
+	// -- just rely on periodic search widening
 	if len(live) > 0 && (s.consecutiveTicks == 0) {
 		s.pm.FindMorePeers(ctx, live[0])
 	}
-	s.resetTick()
+	s.resetIdleTick()
 
 	if len(s.liveWants) > 0 {
 		s.consecutiveTicks++
 	}
 }
 
-func (s *Session) handleRebroadcast(ctx context.Context) {
-
-	if len(s.liveWants) == 0 {
+func (s *Session) handlePeriodicSearch(ctx context.Context) {
```

Comment:
I guess this _is_ now a rebroadcast, but, IMO, periodic search is still more accurate.